portfoliohacktoberfest/index.html

```diff
@@ -0,0 +1,569 @@
+                            Read More
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="background-section-3"></div>
+        </section>
+
+
+        <!-- SECTION NO 4 -->
+        <section class="container-fluid section-4">
+            <div class="row">
+                <div class="col-lg-6">
+                    <div class="section-4-left text-center text-lg-end pe-0 pe-lg-5">
+                        <span class="section-4-heading text__light">We</span>
+                        <span class="section-4-heading text__light">Build</span>
+                        <span class="section-4-heading-bold fw-bolder">Digital</span>
+                    </div>
+                </div>
+                <div class="col-lg-6">
+                    <div class="section-4-right text-center text-lg-start">
+                        <div class="sec4-blank"></div>
+                        <div class="sec4-text">
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus est dolor iure voluptates.
+                            Facere nostrum porro, velit veniam, architecto veritatis, libero a repellat quod quas atque
+                            totam accusamus obcaecati delectus dicta! Suscipit ratione qui, nihil voluptates error
+                            distinctio optio doloremque?
+                        </div>
+                        <div class="section-4-button">
+                            <button class="sec4-btn text-uppercase fw-bold" onclick="location.href='#'">Read
+                                More</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Divider -->
+        <div class="bold-line"></div>
+
+
+        <!-- SECTION NO 5 -->
+        <section id="my-portfolio" class="container-fluid section-5">
+            <div class="row">
+                <div class="col">
+                    <div class="section-5-text text-center mx-auto">
+                        We Design People Inspired Experiences That Create Positive Change In People's Lives.
+                    </div>
+                    <div class="section-5-icon text-center">
+                        <i class="bi bi-chevron-double-down sec5-icon"></i>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col text-center">
+                    <div class="sec5-portfolio-head">
+                        View Portfolio
+                    </div>
+                </div>
+            </div>
+
+            <div class="row mt-4 mt-md-5 work-gallery-row">
+                <div class="masonry">
+                    <!-- Work 1 -->
+                    <div class="grid">
+                        <img src="./images/work-1.jpg" class="img-fluid image-work">
```

Comment:
Please include the images in the folder. I cannot access them as they are accessed inside your local repo.